### PR TITLE
Add ipython example

### PR DESCRIPTION
**Pull request by corradio**
Refers to #265.

Changes:
- cast timestamp values to `pd.Timestamp`
- rename `sources` to `mode`
- rename `value` in exchanges to `net_flow`
- make sure we iterate keys and values in the proper order
- add renewables.ipynb
- add dependencies
- set file names to lowercase

### CODE REVIEW

**Comment by martindaniel4** [comment]
@corradio - looks like you edited most of the changes from your review here - https://github.com/corradio/electricitymap/pull/292

Do you want to merge that one and I close mine ?

**Comment by corradio** [comment]
Let's first merge yours!

**Comment by corradio** [comment]
@martindaniel4 can you review? Feel free to suggest betters ways to do things in my ipynb.
cc @brunolajoie , you should check out the new renewables.ipynb

**Comment by martindaniel4** [comment]
I do not have my computer today. I will take a look when I can!

File: datascience/README.md [hunk 1]
```diff
@@ -69,6 +83,7 @@ Parameters :
 - countries (*list*) - a list of country codes to pull.
 - start_date (*string*) - start date. Date format is `YYYY-MM-DD`.
 - end_date (*string*) - end date. Date format is `YYYY-MM-DD`.
+- delta(*int*) - interval of timestamp between `start_date` and `end_date`. In minutes.
```
Comment (martindaniel4) [inline datascience/README.md]: It looks like the parameter `delta` is missing from the title of the function

File: datascience/utils.py [hunk 2]
```diff
@@ -23,25 +24,30 @@ def fetch_production(country_code, t, delta):
...
+            modes = o['production'].keys()
+            p = pd.DataFrame(
+                data={
+                    'timestamp': pd.Timestamp(t.datetime),
+                    'country': country,
+                    'mode': modes,
+                    'production': map(lambda k: o['production'][k], modes),
+                })
+            if df is not None: df = df.append(p)
+            else: df = p
```
Comment (martindaniel4) [inline datascience/utils.py]: 👍

File: datascience/exchange.ipynb [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+   "execution_count": 6,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/usr/local/lib/python2.7/site-packages/ggplot/ggplot.py:192: RuntimeWarning: Setting \"mpl.rcParams['axes.prop_cycle']=cycler(u'color', [u'#333333', u'#348ABD', u'#7A68A6', u'#A60628', u'#467821', u'#CF4457', u'#188487', u'#E24A33'])\" raised an Exception: u'axes.prop_cycle is not a valid rc parameter.See rcParams.keys() for a list of valid parameters.'\n",
```
Comment (martindaniel4) [inline datascience/exchange.ipynb]: @corradio - I would remove the output of that cell to make sure the warning does not appear on Github

File: datascience/README.md (hunk 1 above)
Comment (corradio) [inline datascience/README.md]: good catch!

File: datascience/exchange.ipynb (hunk 3 above)
Comment (corradio) [inline datascience/exchange.ipynb]: do you know how to remove the output and make sure the plot still appears?

**Comment by corradio** [comment]
@martindaniel4 why do you think we should move the files in a notebook folder? Isn't that too much?

**Comment by martindaniel4** [comment]
@corradio yeah nevermind. Let's merge this!

**Pull request merged**